package.json

```diff
@@ -22,18 +22,18 @@
     "lint:fix": "pnpm run prettier:fix && pnpm run eslint:fix && pnpm run cspell:check && pnpm run typescript:check"
   },
   "dependencies": {
-    "@docusaurus/core": "2.3.0",
-    "@docusaurus/preset-classic": "2.3.0",
+    "@docusaurus/core": "2.3.1",
+    "@docusaurus/preset-classic": "2.3.1",
     "@mdx-js/react": "^1.6.22",
     "clsx": "^1.2.1",
     "prism-react-renderer": "^1.3.5",
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
   },
   "devDependencies": {
-    "@docusaurus/module-type-aliases": "2.3.0",
-    "@docusaurus/theme-classic": "^2.3.0",
-    "@docusaurus/types": "^2.3.0",
+    "@docusaurus/module-type-aliases": "2.3.1",
+    "@docusaurus/theme-classic": "^2.3.1",
+    "@docusaurus/types": "^2.3.1",
```

Comment:
Can you makeall of the docusaurus stuff allow compatible versions? (`^` prefix)

Reply:
Done